docs/tutorial.md

```diff
@@ -0,0 +1,138 @@
+# Tutorial
+
+In this tutorial we will deploy the [sample express Node.js HTTP server](https://expressjs.com/en/starter/hello-world.html) in your own AWS account with AnyCloud. All the code can be found in this [template repository](https://github.com/alantech/hello-anycloud) which you can use to [create a new repository](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template) for your AnyCloud project.
+
+## Enable programmatic AWS access for AnyCloud
+
+1. Create a new an IAM user in your AWS account using their console/UI as described [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html#id_users_create_console).
+2. Create a new access key under that IAM user using their console/UI as described [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey).
+3. Enable programmatic access for that IAM user, and attach the built-in [`AmazonEC2FullAccess`](https://console.aws.amazon.com/iam/home#/policies/arn%3Aaws%3Aiam%3A%3Aaws%3Apolicy%2FAmazonEC2FullAccess)policy to it as described [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_manage-attach-detach.html#add-policies-console).
+4. Take the `accessKeyId` and `secretAccessKey` from step 2 and add a local `~/.anycloud/credentials.json` file with the following contents:
+
+```javascript
+{
+  "aws-personal": {
+    "cloudProvider": "AWS",
+    "credentials": {
+      "accessKeyId": "#####################",
+      "secretAccessKey": "###################"
+    },
+  }
+}
+```
```

Comment:
This bit *really* makes me wish we had an `anycloud config` sub-command with perhaps interactive prompting of what provider, what credentials, and what alias to provide.

Reply:
Yeah, this has been on my mind. Instead of another subcommand. In my head, it would go into the interactive prompting whenever the configuration is not properly set up.